pkgs/development/python-modules/opentelemetry-util-http/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, hatchling
+, opentelemetry-instrumentation
+, opentelemetry-sdk
+, opentelemetry-semantic-conventions
+, opentelemetry-test-utils
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "opentelemetry-util-http";
+  version = "0.38b0";
```

Comment:
Is this a beta release? Can we use a none beta release?

Reply:
There are only beta releases of this package: https://pypi.org/project/opentelemetry-util-http/#history